src/NOTAM/NOTAMGlue.cpp

```
@@ -0,0 +1,1520 @@
+    } else if (!load_committed) {
+      retry_pending = false;
+    } else {
+      retry_pending = false;
+      fetch_failure_notified = false;
+    }
+  }
+
+  if (start_queued_refresh) {
+    LogFmt("NOTAM: Starting queued forced refresh");
+    Cancel();
+    load_task.Start(LoadNOTAMsInternal(queued_location, true),
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function declared not to throw exceptions.

(throwInNoexceptFunction)